src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -133,9 +139,39 @@ public PersonBuilder withNoEmergencyContact() {
     }
 
     /**
-     * Builds a person using AB3 constructor.
+     * Sets the {@code GradYear} of the {@code Person} that we are building to null.
```

Comment:
Should the comment here be sets the GradYear to the specified grad year instead of null?